src/cop1.c

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file COP1.c
+ * @brief N64 COP1 Interface
+ * @ingroup n64sys
+ */
+#include "libdragon.h"
+
+/**
+ * @addtogroup n64sys
+ * @{
+ */
+
+__attribute__((constructor)) void __init_cop1()
+{
+    /* Read initalized value from cop1 control register */
+    uint32_t fcr31 = C1_FCR31();
+    
+    /* Set FS bit to allow flashing of denormalized floats */
+    fcr31 |= C1_FS_BIT;
```

Comment:
I think you should elaborate a little bit on this. I personally don't know what means "flashing a denomal", so I'd explain here (or a in a function-level comment) why this is required and how it is good.